_sass/elements/_buttons.scss

```diff
@@ -150,6 +150,10 @@ button,
     font-size: 1.9rem;
     padding: 1.5rem 3rem;
   }
+
+  &.usa-text-capitalize {
```

Comment:
Is this class name something that comes from U.S. web design standards or is it something we created? If the latter, I don't think we should prefix it with `usa`, since I think that's reserved for U.S. web design standards classes.